tests/test_plugins/discovery_test_plugin/hydra_plugins/discovery_test/__not_hidden_plugin.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+from hydra.plugins.plugin import Plugin
+
+
+class NotHiddenTestPlugin(Plugin):
+    def __init__(self, v: int) -> None:
+        self.v = v
+
+    def add(self, x: int) -> int:
+        return self.v + x
```

Comment:
You can remove the add impl. just have an empty plugin without any bells and whistles.
```python
class NotHiddenTestPlugin(Plugin):
  ... 
```